elcord.el

```diff
@@ -121,6 +121,11 @@ The mode text is the same found by `elcord-mode-text-alist'"
   :type 'boolean
   :group 'elcord)
 
+(defcustom elcord--show-small-icon 't
```

Comment:
Can you name this with a single hyphen?
Convention is that any internal functions to a package should have two hyphens, and externals (like customization vars) should have only one hyphen.
Thanks!